app/src/main/java/io/apicurio/registry/auth/AuthorizedInterceptor.java

```diff
@@ -119,6 +119,11 @@ public Object authorizeMethod(InvocationContext context) throws Exception {
             return context.proceed();
         }
 
+        // If Authenticated read access is enabled, and the operation is read, allow it.
+        if (authConfig.authenticatedReadAccessEnabled && annotation.level() == AuthorizedLevel.Read) {
```

Comment:
I think this should **also** check `AuthorizedLevel.None`.  So if the level is Read **or** None, then it's OK.

Reply:
Yup, you're right.